Week5/csv write.py

```diff
@@ -1,46 +1,8 @@
-#!/usr/bin/env python
-# coding: utf-8
-
-# In[7]:
-
-
 import csv
-
-
-# In[8]:
-
-
 f=open('c:\\sqlite3\\csv\\student.csv','w',newline='')
```

Comment:
is there any other method to open file ?

Reply:
okay
what is the difference between open() and with open()  ?

Reply:
in open() file have to close()
in with open() file automatically close()

Reply:
okay